### DISCUSSION TITLE
internlm/l0-linux/

### DISCUSSION THREAD

**Discussion by giscus** [x2rs/x2rs.github.io#22] [Giscus Comments]
# internlm/l0-linux/

My personal documentation website.

https://x2rs.github.io/internlm/l0-linux/

**Comment by Luchen-0420** [comment]
1.关于helloworld.py这个作业，最好是按照文档内的做一遍，主要涉及到端口映射步骤。

**Reply by zoyuki11** [reply]
感谢助教大大的指出，我重新按照文档完成了这部分内容。之前做的时候理解错了 :cry: ，以为只要跑个 Hello World 就行了qwq
